File: Persegi (Stroke)/Java/persegi-stroke-i.java
import java.util.Scanner;

public class Main{
	public static void main(String[] args) {
	    
	    Scanner userInput = new Scanner(System.in);
    	
        System.out.print("Masukan Lebar Persegi : ");
        int x = userInput.nextInt();
        
        for(int j = 0; j <= x; j++){
            System.out.print("* ");
        }
        System.out.println("");
        
        for(int i = 0; i <= (x-2); i++){
            System.out.print("* ");
            for(int j = 0; j <= (x-2); j++){
                System.out.print("  ");
            }
            System.out.println("*");
        }
        
        for(int k = 0; k <= x; k++){
            System.out.print("* ");
        }
	}
}

File: Persegi (Stroke)/C++/persegi-stroke-i.cpp
#include <iostream>
using namespace std;

int main(){
    
    int x;
    cout << "Masukan Lebar Persegi : ";
    cin >> x;
    
    for(int j = 0; j <= x; j++){
        cout << "* ";
    }
    cout << endl;
    
    for(int i = 0; i <= (x-2); i++){
        cout << "* ";
        for(int j = 0; j <= (x-2); j++){
            cout << "  ";
        }
        cout << "*";
        cout << endl;
    }
    
    for(int k = 0; k <= x; k++){
        cout << "* ";
    }

    return 0;
}
